shutter/low-level.md

```diff
@@ -219,15 +274,19 @@ def extract_message_parts(message: bytes) -> Tuple[bytes, uint64, Address, uint6
     nonce_bytes = prefix[37:45]
 
     chain_id = int.from_bytes(chain_id_bytes, "big")
+    assert chain_id >= 0
```

Comment:
I don't know if this is required in the python pseudocode or if the uint typehint is enough. 
The `uint64` typehint doesn't exist in python, but I saw it used in other ethereum pseudocode as well.

Reply:
removed this due to the uint64 type hint